storage/vc_aos/include/algebra/storage/vc_aos.hpp

```diff
@@ -90,6 +90,16 @@ struct index<Vc_1::Vector<T, Vc_1::simd_abi::fixed_size<N>>> {
   using size_type = vc_aos::size_type;
 };
 
+template <typename T, auto N>
+struct value<vc_aos::storage_type<T, N>> {
+  using type = T;
+};
+
+template <typename T, auto N>
+struct value<Vc_1::Vector<T, Vc_1::simd_abi::fixed_size<N>>> {
+  using type = T;
+};
```

Comment:
Would it make sense to template this on a typename instead of an integer `N`?

Reply:
You mean the `value` struct? The other type `Vc_1::Vector<T, Vc_1::simd_abi::fixed_size<N>>` is not not part of algebra plugins

Reply:
No what I mean is to replace this by:

```suggestion
template <typename T, typename Q>
struct value<Vc_1::Vector<T, Q>> {
  using type = T;
};
```

Reply:
Ah, Ok! Makes sense, let me try

Reply:
My guess is, that fixed size is the abi for the SimdArray (which has a fixed number of elements rather than a fixed vector register type). Using it might be more explicit compared to the SoA plugin